src/MonitoringTargetStage.ts

```diff
@@ -1,18 +1,22 @@
 import path from 'path';
-import { aws_sns, aws_ssm, Duration, Stack, StackProps, Stage, StageProps, Tags } from 'aws-cdk-lib';
+import { aws_sns, aws_ssm, Duration, Environment, Stack, StackProps, Stage, StageProps, Tags } from 'aws-cdk-lib';
 import { Runtime } from 'aws-cdk-lib/aws-lambda';
 import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';
 import { LambdaSubscription } from 'aws-cdk-lib/aws-sns-subscriptions';
 import { Construct } from 'constructs';
 import { Statics } from './statics';
 
+export interface MonitoringTargetStageProps extends StageProps {
+  deployToEnvironments: { name: string; env: Environment }[];
```

Comment:
Deze wordt nog niet gebruik in deze stage?

Reply:
Scherp, ik was 't aan het ombouwen maar bedacht me over 1 stuk. Ga ik eerst even omzetten en in de dev-branch uitvoeren.